lib/oxidized/model/tplink.rb

```diff
@@ -31,7 +31,11 @@ class TPLink < Oxidized::Model
   end
 
   cmd 'show system-info' do |cfg|
-    comment cfg.each_line.to_a[3..-3].join
+    cfg = cfg.each_line.to_a[1..-3].join
+    cfg.gsub! /^ System\ Time\s.+/, '' # Omit constantly changing uptime info
+    cfg.gsub! /^ Running\ Time\s.+/, '' # Omit constantly changing uptime info
+    cfg.gsub! /^\S+/, ''
```

Comment:
What does this remove?

Reply:
It removes runtime and system time from output, as it is constantly changing. See also https://github.com/ytti/oxidized/pull/1443#issuecomment-429977314

Reply:
Sorry, my question was too unspecific: What does the third `gsub!` line remove? Why should we remove all lines without any whitespace?

Reply:
And probably related: Why do you change the line indices from `[3..-3]` to `[1..-3]`?

My guess is that the latter is correct for most models but includes the prompt under some circumstances, which you want to remove with the last gsub.

Reply:
To answer my own question: The changed indices seem to include these lines:
```
!  System Description   - JetStream 8-Port Gigabit Smart PoE Switch with 2 SFP Slots
!  System Name          - $hostname
```